CODEOWNERS

```diff
@@ -5,8 +5,17 @@
 # reviewed by someone else.
 
 # Plugin-level rules.
+packages/camera/**                                       @bparrishMines
```

Comment:
@bparrishMines I gave you `camera` since you have previous experience there; let me know if that works for you.

Reply:
That works for me!